BootloaderCommonPkg/Library/ContainerLib/ContainerLib.c

```diff
@@ -230,6 +230,28 @@ LocateComponentEntryFromContainer (
   return CompEntry;
 }
 
+/**
+  This function returns the hash alg type from auth type.
+
+  @param[in] AuthType    Authorization Type.
+
+  @retval         Hash Algorithm Type.
+
+**/
+UINT8 GetHashAlg(AUTH_TYPE AuthType){
+  UINT8 HashAlg;

```

Comment:
Why not return HASH_ALG_TYPE instead of UINT8?

Reply:
I would do it.